src/patternfly/components/Card/examples/Card.md

```diff
@@ -252,6 +252,48 @@ cssPrefix: pf-c-card
 {{/card}}
 ```
 
+### Expandable (collapsed)
+```hbs
+{{#> card card--id="card-expandable-example"}}
+  {{#> card-header}}
+    {{> card-header-toggle}}
+    {{#> card-actions}}
+      {{#> dropdown id=(concat card--id "-dropdown-kebab-right-aligned") dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+      {{/dropdown}}
+      <input type="checkbox" id="{{card--id}}-check" name="{{card--id}}-check" aria-labelledby="{{card--id}}-title">
+    {{/card-actions}}
+    {{#> card-title card-title--attribute=(concat 'id="' card--id '-title"')}}
+      Title
+    {{/card-title}}
+  {{/card-header}}
```

Comment:
This example omits `{{#> card-expandable-content}}`, which I think is fine if we move in that direction, however all (or most) other `expandable` things are `hidden` when collapsed. It would be more consistent with other componentry if this where `display: none; visibility: hidden;` and `hidden`.

Reply:
Yeah, I did that specifically here because the card CSS looks for adjacent elements to apply spacing. I'm in favor of setting up our examples like this as a way to indicate to react how the element should be hidden. WDYT @christiemolloy?

Reply:
@jessiehuff Can you see any accessibility issue here? ^^

Reply:
@mattnolting I can see the pros and cons to this approach. I know that we've done this in other components like the dropdown, but I'm concerned that this might have caused the dropdown issue we noticed in JAWS. We created a workaround by adding additional key listeners to shift focus, but I suspect that since the content didn't exist before JAWS opened the dropdown, it didn't recognize it and announce it. So I suppose it's something to be on the lookout for when we use this approach. I'm guessing that whenever react implements it, we'll just need to test to make sure it doesn't have the same issue. @mcoker also had an idea of how to test out this theory as well to see if it was the culprit. 🙂